File: PRACTICA DE CAMPO 01-S02/C#/S2_PRACTICACAMPO/s2_clasificador_notas.cs
Console.WriteLine("Ingrese nota para clasificar:");
double nota= double.Parse(Console.ReadLine());
if (nota<0 || nota>20){
    Console.WriteLine("Nota no valida solo se aceptan valores entre 0 y 20.");
}else{
    if (nota==20){
        Console.WriteLine("Excelente.");
    }else if (nota>=11){
        Console.WriteLine("Aprobado.");
    }else{
        Console.WriteLine("Desaprobado.");
    }
}

File: PRACTICA DE CAMPO 01-S02/PYTHON/s2_clasificador_notas.py
nota=float(input("Ingrese nota para clasificar:"))
if nota<0 or nota>20:
    print("Nota no valida solo se aceptan valores entre 0 y 20.")
else:
    if nota==20:
        print("Excelente.")
    elif nota>=11:
        print("Aprobado.")
    else:
        print("Desaprobado.")
